Source/Core/DolphinWX/GameListCtrl.h

```diff
@@ -117,4 +125,6 @@ class CGameListCtrl : public wxListCtrl
   int last_sort;
   wxSize lastpos;
   wxEmuStateTip* toolTip;
+
+  std::vector<ColumnInfo> columns;
```

Comment:
`ColumnInfo` can be a forward declaration. This variable in particular should also have an `m_` prefix.
 
e.g.

```cpp
// Header
class CGameListCtrl
{
public:
    ...
private:
    struct ColumnInfo;

    void ReloadList();

    ...

    std::vector<ColumnInfo> m_columns;
};

// cpp file
// Placed anywhere before its first usage (in this case before the constructor).
struct CGameListCtrl::ColumnInfo
{
    const int id;
    const int default_width;
    const bool resizable;
    bool& visible;
};
```